components/Ui/UiCard.vue

```diff
@@ -93,6 +94,7 @@ interface Props {
   ctaLabel?: string;
   image?: string;
   imageContain?: boolean;
+  imageDescription?: string;
```

Comment:
Try to use the same name through all the pages, I have seen `imageDescription`, `altText` and `alt` to refer to the same thing